releasenotes/notes/1.0/prepare-1.0-e13b9b8273055775.yaml

```diff
@@ -0,0 +1,70 @@
+---
+prelude: |
+  We are very pleased to release Qiskit 1.0.0. This release is the
+  culmination of 7 years of development to mature Qiskit into a stable high
+  performance SDK to perform quantum computation, and is the start of new
+  era for the Qiskit project and community. The most significant changes with the 1.0.0 
+ release relate to the stability policy, release cycles, and versioning.
+  Qiskit is now formally using `semantic versioning <https://semver.org/>`__ ,
+  and this means that, for the entire lifecycle of the 1.x release series, the project
+  is commited to maintaining backwards compatibility. We're also now starting
```

Comment:
I think one more sentence about backwards compatibility would be nice here: do we commit to only reading the output (e.g. quantum circuits) written by earlier Qiskit versions or do we commit to some API, i.e. guarantee that user code will run correctly for the entire lifecycle of a major release?

Reply:
I'll come up with something. I was trying to avoid rehashing the entire contents of the documentation on this topic (which is what I ended up doing in my initial draft), but I definitely can see the value in explaining a bit more about what backwards compatibility means here.